### ISSUE TITLE
[Question] Can CLIENT REPLY OFF be used

### ISSUE THREAD

**Issue by gcleaves** [luin/ioredis#1245]
Is there a way to get ioredis to work with CLIENT REPLY OFF ?

**Comment by gcleaves** [comment]
Yello?

**Comment by luin** [comment]
Hi Geoffrey,
Sorry for the late response! I didn't notice this one. Currently, it's not supported but we do have a plan for it. What use case do you want to use that command?

**Comment by gcleaves** [comment]
Hi @luin ,  I need to load about 20 million records into Redis in a daily batch process. I use pipelines to bucket 100K records at a time, then exec. I don't really care about the response. https://github.com/gcleaves/ip_denylist_lookup/blob/837fffb3390aee13f610ddc26f50fe1d467a362a/loadToRedis.js#L92

I think this is the correct use case for CLIENT REPLY OFF, let me know if I am mistaken.

Thanks.

**Comment by luin** [comment]
Yes, that's a perfectly valid use case for `CLIENT REPLY OFF`. Thanks for sharing!
